manager/main.go

```diff
@@ -84,21 +85,40 @@ func run(namespace string, metricsAddr string, enableLeaderElection bool,
 
 	setupLog.Info().Msg("Manager client rate limits: qps = " + fmt.Sprint(client.QPS) + " burst=" + fmt.Sprint(client.Burst))
 
+	// Set health probes address(required to run probes)
+	// and desired liveness and readiness endpoints(optional)
 	mgr, err := ctrl.NewManager(client, ctrl.Options{
-		CertDir:            environment.GetDataDir() + certSubDir,
-		Scheme:             scheme,
-		Namespace:          namespace,
-		MetricsBindAddress: metricsAddr,
-		LeaderElection:     enableLeaderElection,
-		LeaderElectionID:   "fybrik-operator-leader-election",
-		Port:               controllers.ManagerPort,
-		NewCache:           cache.BuilderWithOptions(cache.Options{SelectorsByObject: selectorsByObject}),
+		CertDir:                environment.GetDataDir() + certSubDir,
+		Scheme:                 scheme,
+		Namespace:              namespace,
+		MetricsBindAddress:     metricsAddr,
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "fybrik-operator-leader-election",
+		Port:                   controllers.ManagerPort,
+		HealthProbeBindAddress: healthProbeAddr,
+		LivenessEndpointName:   "health",
```

Comment:
these are the default values you don't have to set them. 
However if you set, they should be `/readyz` `/healthz` 

Now curl is working.